integration/remote/util/util_windows.go

```diff
@@ -110,9 +110,10 @@ func parseEndpoint(endpoint string) (string, string, error) {
 		return "", "", err
 	}
 
-	if u.Scheme == "tcp" {
+	switch u.Scheme {
+	case "tcp":
```

Comment:
Ah, right, this one is re-doing the other change, so probably should be a squashed with the other commit (or order of commits changed)